### DISCUSSION TITLE
Welcome to gopay Discussions!

### DISCUSSION THREAD

**Discussion by iGoogle-ink** [go-pay/gopay#194] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zeqi0607** [comment]
最近支付宝上了“订阅”支付新功能，能否支持一下，支付宝订阅支付文档：https://opendocs.alipay.com/pre-open/0ixmsx?pathHash=82f7e177
